upgrade/lib/validation/CASMREL-776-CSM12-NCN-boot-order-backport/scripts/mini-install.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Author: Russell Bunch <doomslayer@hpe.com>
+trap "printf >&2 'Metal Install: [ % -20s ]' 'failed'" ERR TERM HUP INT
+trap "echo 'See logfile at: /var/log/cloud-init-metal.log'" EXIT
+set -e
+
+# Echo that we're the original, but stripped down script.
+echo "Running stripped CSM 1.2 install.sh script, $0"
+# See the original "install.sh" here: https://raw.githubusercontent.com/Cray-HPE/node-image-build/develop/boxes/ncn-common/files/scripts/metal/install.sh?token=AB3JQE4CBRQKCKMZ5UAKHH3BL4264'
```

Comment:
what is this token?  I see the security scanner flagged it...

Reply:
It's a permalink, and it's wrong.